modules/contentbox/modules/contentbox-admin/handlers/authors.cfc

```diff
@@ -354,7 +360,7 @@ component extends="baseHandler"{
         var vResults = validateModel( target = prc.oAuthor, excludes = "password" );
 		if ( vResults.hasErrors() ) {
             cbMessagebox.warn( messageArray=vResults.getAllErrors() );
-            flash.keep( "layout,xehInvalidData,xehEnrollmentSuccess,xehEnrollmentSuccessQueryString" );
+            flash.keep( "layout,xehInvalidData,xehEnrollmentSuccess,xehEnrollmentSuccessQueryString" )
```

Comment:
At this point, just do a `flash.keep()` with not items, this will keep all flashed state.

Reply:
done